validator/src/main.rs

```diff
@@ -735,6 +735,14 @@ pub fn main() {
                 .validator(hash_validator)
                 .help("Require the genesis have this hash"),
         )
+        .arg(
+            Arg::with_name("expected_bank_hash")
```

Comment:
How about making `ledger-tool create-snapshot` output the bank hash, like it outputs the shred version right now.  Otherwise seems like you'd need to trudge through logs to find the expected bank hash?

Reply:
I can add that. You would have to see this log:
```
[2020-06-30T17:39:39.140958185Z INFO  solana_runtime::bank] bank frozen: 21042873 hash: 6q2oTgs8FiJxr
a2Zo1N8tzWqo5b6uGbjmFgoWDsXxchY accounts_delta: 5wfkgiJFPFiNWmijnoC4CeJzhUpzrLHgmXXVkWkrdBYL signatur
e_count: 32 last_blockhash: E9umzWf1Hb7TZ1Ne8bR2d9saFHtxAwZMQ3E5qKvwb84j capitalization: 100614924490
42218618
```
But it is not super obvious. As you said you have to look for it a bit.

Reply:
Added.